src/main/mocha.ts

```diff
@@ -98,6 +100,9 @@ export default class MochaWrapper extends Mocha {
 
     taskManager
       .on('taskFinished', (testResults: ISubprocessResult) => {
+        if (!testResults) {
+          throw new Error('No output from test');
+        }
```

Comment:
I'm not convinced that we need this behaviour:

1) `mocha` doesn't have it so if it happens this will be something weird for users
2) it's not covered with a test

Reply:
Removed